app/javascript/widget/views/Unread.vue

```diff
@@ -99,7 +99,6 @@ export default {
   display: flex;
   flex-direction: column;
   flex-wrap: nowrap;
-  justify-content: flex-end;
```

Comment:
on IE it seems the UI was pushed to the bottom of its container out of view of the screen.